packages/react-data-grid/src/Cell.tsx

```diff
@@ -159,31 +160,96 @@ export default class Cell<R> extends React.Component<CellProps<R>> implements Ce
     return allEvents;
   }
 
+  handleDeleteSubRow = (): void => {
+    const { idx, rowIdx, expandableOptions, cellMetaData, rowData } = this.props;
+
+    const { onDeleteSubRow } = cellMetaData;
+
+    if (onDeleteSubRow) {
+      onDeleteSubRow({
+        idx,
+        rowIdx,
+        rowData,
+        expandArgs: expandableOptions
+      });
+    }
+  }
+
   render() {
-    const { idx, rowIdx, column, value, tooltip, children, height, cellControls, expandableOptions, cellMetaData, rowData, isScrolling } = this.props;
+    const { rowIdx, column, value, tooltip, children, height, cellControls, expandableOptions, cellMetaData, rowData, isScrolling, isSummaryRow } = this.props;
     if (column.hidden) {
       return null;
     }
 
     const style = this.getStyle();
-    const className = this.getCellClass();
-    const cellContent = children || (
-      <CellContent<R>
-        idx={idx}
-        rowIdx={rowIdx}
-        column={column}
-        rowData={rowData}
-        value={value}
-        tooltip={tooltip}
-        expandableOptions={expandableOptions}
-        height={height}
-        onDeleteSubRow={cellMetaData.onDeleteSubRow}
-        cellControls={cellControls}
-        isScrolling={isScrolling}
-        isRowSelected={this.props.isRowSelected}
-        onRowSelectionChange={this.props.onRowSelectionChange}
+    const cellClassName = this.getCellClass();
+    const cellValueClassName = classNames('react-grid-Cell__value', { 'cell-tooltip': !!tooltip });
+
+    if (isSummaryRow) {
+      return (
+        <div
+          ref={this.cell}
+          className={cellClassName}
+          style={style}
+        >
+          <div className={cellValueClassName}>
+            <div className="react-grid-Cell__container">
+              <span>
+                <CellValue<R>
```

Comment:
We can do something like 

```
const cellValue = (<CellValue<R> ... />);


if (isSummaryRow) {
 ....
   <div className="react-grid-Cell__container">
      {cellValue}
   </div>
```

Reply:
Now I think about it, I think it is better to use `CellValue` for summary rows instead of `CellContent`. This way we do not have to move the `Expand/Collapse` functionality here